test/util/cluster/cluster.go

```diff
@@ -308,5 +309,10 @@ func GetNodeSecureBootState(clients *testclient.ClientSet, nodeName, namespace s
 		return false, err
 	}
 
+	err = clients.Pods(namespace).Delete(context.Background(), runningPod.Name, metav1.DeleteOptions{GracePeriodSeconds: pointer.Int64Ptr(0)})
```

Comment:
should we use `defer` to delete the pod in case of early return ?

Reply:
@SchSeba  WDYT ? :)

Reply:
sorry for the late response :)